### DISCUSSION TITLE
Extras Addon on yimmenu

### DISCUSSION THREAD

**Discussion by mateoszo** [Deadlineem/Extras-Addon-for-YimMenu#145] [Q&A]
it Extras Addon made only for Chronix? I tried to use extras lua on yimmenu only but it didint work i have the extras addon 1.0 but i wanted to use newer version on yimmenu beacuse i have it set up already.

**Comment by Deadlineem** [comment]
> it Extras Addon made only for Chronix? I tried to use extras lua on yimmenu only but it didint work i have the extras addon 1.0 but i wanted to use newer version on yimmenu beacuse i have it set up already.

When you try to use it, does it show errors in the console?  It should work for any version of YimMenu including tupoy and chronix

**Reply by mateoszo** [reply]
> > it Extras Addon made only for Chronix? I tried to use extras lua on yimmenu only but it didint work i have the extras addon 1.0 but i wanted to use newer version on yimmenu beacuse i have it set up already.
> 
> When you try to use it, does it show errors in the console? It should work for any version of YimMenu including tupoy and chronix

Ok so i disabled all my scripts in the image but it says 0 gui and scripts and my console is in second image. All my other scripts work but not extra addon but the old one 1.0ver works on yimmenu.
![image](https://github.com/user-attachments/assets/72248d0a-cadc-454b-9c5f-064615b2cc60)

![image](https://github.com/user-attachments/assets/601e2ae8-cdeb-4b36-9a58-1377bb811036)
